src/components/shared/basic/Dropzone/index.tsx

```diff
@@ -111,6 +112,25 @@ export const Dropzone = ({
     [currentFiles, onChange, handleDelete]
   )
 
+  const handleFileSizeValidator = (file: File) => {
+    const formatFileSize = (bytes: number): string => {
+      const sizeInMB = bytes / CONVERT_TO_MB
+      if (Number.isInteger(sizeInMB)) {
+        return sizeInMB.toString()
+      } else {
+        return sizeInMB.toFixed(1)
+      }
+    }
+
+    if (!maxFileSize) return null
+    return file.size > maxFileSize
+      ? {
+          code: 'size-too-large',
+          message: `File is larger than ${formatFileSize(maxFileSize)} MB`,
```

Comment:
Please translate messages in frontend components. See also my comment: https://github.com/eclipse-tractusx/portal-frontend/pull/1185#discussion_r1802580964

Reply:
I have updated the translation in PR: https://github.com/eclipse-tractusx/portal-frontend/pull/1185.
Since the translation would be the same so changed in above PR to avoid merge conflict for the same thing. It would be better if you can review and I merge the mentioned PR(https://github.com/eclipse-tractusx/portal-frontend/pull/1185) first.